src/TimeRestricted.sol

```diff
@@ -36,15 +36,55 @@ import {BytesHelper} from "src/BytesHelper.sol";
 /// and the actual module addresses are the same
 /// check that the owners are the same
 
+/// Blocks all delegate calls, as the owners and modules could be changed.
+/// Does not allow changing of the implementation contract either through
```

Comment:
I looked at Safe contracts and could not find a method through which implementation contract could be changed via a normal safe transaction. Is there a method for this on safe?

Reply:
I understand that this could be done via a delegate call but was confused how this can be done via a normal safe transaction. If we are blocking delegate calls then do we need to check that implementation was unchanged via transient storage.